app/overrides/spree/admin/shared/_configuration_menu.rb

```diff
@@ -1,7 +1,7 @@
 Deface::Override.new(
```

Comment:
Is there a way to add these without using a Deface?

Reply:
I'm going to remove these overrides and if we need them, I can create another PR including them, thanks